sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md

```diff
@@ -4,9 +4,6 @@
 
 **Breaking Changes**
 
-- `create_data_feed` now takes as input a `DataFeed` object
-- `create_metric_anomaly_detection_configuration` now takes as input an `AnomalyDetectionConfiguration` object
-- `create_anomaly_detection_configuration` now takes as input an `AnomalyAlertConfiguration` object
 - `create_hook` now takes as input an `EmailHook` or `WebHook`
```

Comment:
```suggestion
```

Reply:
Actually will we make the change to create_hook as well? It used to take the name parameter separate from the object.

Reply:
It seems to me the difference is
to take name+hook object -> to hook object only.

I don't think we should revert back to name+object.